src/App.tsx

```diff
@@ -2,34 +2,60 @@ import { useEffect, useState } from "react";
 import { Flex } from "@mantine/core";
 import MenuBar from "./Pages/HomePage/components/MenuBar/MenuBar";
 import { useDisclosure } from "@mantine/hooks";
+import dayjs from "dayjs";
 
 import CreateTask from "./Pages/HomePage/components/CreateTask/CreateTask";
 import DisplayTask from "./Pages/HomePage/components/DisplayTask/DisplayTask";
-import { getTasksFromLocalStorage } from "./localstorage/localstorage";
+import { getSortedTasks } from "../src/Shared/Utils/localstorage";
 import { Task } from "./types/Task";
 
 function App() {
   const [opened, { open, close }] = useDisclosure(false);
   const [tasks, setTasks] = useState<Task[]>([]);
+  const [displayTasks, setDisplayTasks] = useState<Task[]>([]);
```

Comment:
I'm not sure I understand what this is for.

Aren't you tracking the tasks already in the tasks state?